projectRouter.js

```diff
@@ -25,14 +38,38 @@ router.put('/:id', validateProject, validateProjectId, (req,res)=>{
     .catch(()=>{res.status(500).json({message:"error in updating project to database"})})
 })
 
-router.get('/:id', validateProjectId, (req,res)=>{
+//delete project of current id
+router.delete('/:id', validateProjectId, (req,res)=>{
+    projectDB.remove(req.params.id)
+    .then(()=>{res.status(200).json({message:"that project has been removed"})})
+    .catch(()=>{res.status(500).json({message:"there was an error in removing that project form the database"})})
+})
+
+//action get to current id project
+router.get('/:id/actions', validateProjectId, (req,res)=>{
     projectDB.get(req.params.id)
     .then(project =>{
-        res.status(200).json(project)
+        if(project.actions.length>0){
+            res.status(200).json(project.actions)
+        }else {res.status(400).json({message:"this project does not have any actions"})}
     })
     .catch(()=>{res.status(500).json({message:"error in getting project by that ID"})})
 })
 
+//action post to current id project
+router.post('/:id/actions',validateProjectId, validateAction,(req,res)=>{
+    const {id} = req.params;
+    let newAction = req.body;
+    newAction.project_id = id;
+
+    actionsDB.insert(req.body)
+    .then(action=>{res.status(201).json(action)})
+    .catch(()=>{res.status(500).json({message:"error in posting new action to database"})})
```

Comment:
If you wanted to respond with the system error, as well as your own error message, you could structure your .catch this way:

```
 .catch(error => {
      res.status(500).json({
        message: "Error in posting new action to database",
        error: `${error}`
      });
    });
```